agent/xds/clusters.go

```diff
@@ -65,72 +67,55 @@ func makeAppCluster(cfgSnap *proxycfg.ConfigSnapshot) (*envoy.Cluster, error) {
 		}
 		c = &envoy.Cluster{
 			Name:           LocalAppClusterName,
-			ConnectTimeout: 5 * time.Second,
+			ConnectTimeout: time.Duration(cfg.LocalConnectTimeoutMs) * time.Millisecond,
 			Type:           envoy.Cluster_STATIC,
-			// API v2 docs say hosts is deprecated and should use LoadAssignment as
-			// below.. but it doesn't work for tcp_proxy target for some reason.
-			Hosts: []*envoycore.Address{makeAddressPtr(addr, cfgSnap.Proxy.LocalServicePort)},
-			// LoadAssignment: &envoy.ClusterLoadAssignment{
-			//  ClusterName: LocalAppClusterName,
-			//  Endpoints: []endpoint.LocalityLbEndpoints{
-			//    {
-			//      LbEndpoints: []endpoint.LbEndpoint{
-			//        makeEndpoint(LocalAppClusterName,
-			//          addr,
-			//          cfgSnap.Proxy.LocalServicePort),
-			//      },
-			//    },
-			//  },
-			// },
+			// // API v2 docs say hosts is deprecated and should use LoadAssignment as
+			// // below.. but it doesn't work for tcp_proxy target for some reason.
+			// Hosts: []*envoycore.Address{makeAddressPtr(addr, cfgSnap.Proxy.LocalServicePort)},
```

Comment:
Is there a reason this is commented out instead of deleted?

Reply:
Good catch, It was experimental but it seems to work just fine in all the integration tests now so I guess it's good as it is.